src/bundles/page-template.js

```diff
@@ -140,7 +140,7 @@ function PageTemplate({ children, fullScreen, headerActions, alerts }) {
       <header className={pageHeaderAreaClasses}>
         <div className="PageTemplate-header">
           <div className="PageTemplate-headerContent PageTemplate-headerContent--left">
-            {!fullScreen && <Logo />}
+            {!fullScreen && <SiteLogo size="small" />}
```

Comment:
There would be no particular class for this one.

Reply:
What do you mean? Is there styling that needs to go here?

Reply:
I would guess there should be a class SiteLogo--small, yes. Right now you'd set the class and it would not put the normal class by omission, if I'm not mistaking.

Reply:
Oh I see, I made these changes in `DefaultSiteLogo` but copied the old code here 🙈 
Updated!